02_oop_part_1/usage.ts

```diff
@@ -0,0 +1,39 @@
+// Create the inventory
+import {Inventory} from "./code/src/Inventory";
+import {Item} from "./code/src/Item";
+import {Sword} from "./code/src/Sword";
+import {ItemWeightComparator} from "./code/src/ItemWeightComparator";
+import {Pizza} from "./code/src/Pizza";
+
+const inventory: Inventory = new Inventory();
+
+// Create a set of items
+const a: Sword = new Sword(30.4219, 0.7893, 300, 2.032);
```

Comment:
@serhii-yaremchuk-epam, you have missed the `use` method inside the `Item` class. That's why you can't use the `use` method if `a` is `Item`.